src/shared/proto/messages_robocup_ssl_detection_tracked.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto2";
+
+message Vector2 {
+    required float x = 1;
+    required float y = 2;
+}
+
+message Vector3 {
+    required float x = 1;
+    required float y = 2;
+    required float z = 3;
+}
+
+// A single tracked ball
+message TrackedBall {
+    // The quality of the detection
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float quality = 1;
+
+    // The position (x, y, height) [m] in the ssl-vision coordinate system
+    required Vector3 pos = 2;
+
+    // The velocity [m/s] in the ssl-vision coordinate system
+    optional Vector3 vel = 3;
+
+    // The initial (kick) speed [m/s], if the ball is moving
+    optional Vector3 initial_speed = 4;
+    // The time [s] until the ball will come to a stop
+    optional float stop_time = 5;
+    // The position [m] at which the ball will come to a stop
+    optional Vector2 stop_pos = 6;
+}
+
+// A single tracked robot
+message TrackedRobot {
+    // The team color of the robot
+    enum TeamColor {
+        UNKNOWN = 1;
+        YELLOW = 2;
+        BLUE = 3;
+    }
+
+    // The robot number
+    required uint32 id = 1;
+    // The team color
+    required TeamColor team_color = 2;
+
+    // The quality of the detection
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float quality = 3;
+
+    // The x-position [m] in the ssl-vision coordinate system
+    required float pos_x = 4;
+    // The y-position [m] in the ssl-vision coordinate system
+    required float pos_y = 5;
+    // The orientation [rad] in the ssl-vision coordinate system
+    required float orientation = 6;
+
+    // The x-velocity [m/s] in the ssl-vision coordinate system
+    optional float vel_x = 7;
+    // The y-velocity [m/s] in the ssl-vision coordinate system
+    optional float vel_y = 8;
+    // The angular velocity [rad/s] in the ssl-vision coordinate system
+    optional float vel_angular = 9;
+
+    // The robot height
+    required float height = 10;
+}
+
+// A frame that contains all currently tracked objects on the field
+message TrackedFrame {
```

Comment:
Is this for the entire field, by merging across all cameras, or for a single camera? Some clarification of the intent in the comments would help.

Reply:
I took it for granted, that there is no separation between different cameras, but I added it to the comment now to make this clear.
The whole idea of the protocol is, that the receiver does not need to care about cameras at all ;)